index.html

```diff
@@ -178,30 +190,47 @@
         rules in this section apply.
       </p>
       <pre class="idl">
+        enum CaptureFacingMode { "user", "environment" };
+
         partial interface HTMLInputElement {
-            [CEReactions] attribute boolean capture;
+            [CEReactions] attribute CaptureFacingMode capture;
         };
       </pre>
       <div>
         <p>
-          The <code><a>capture</a></code> attribute is a <a>boolean
-          attribute</a> that, if specified, indicates that the capture of media
-          directly from the device's environment using a <a>media capture
-          mechanism</a> is preferred.
+          The <dfn data-dfn-for="CaptureFacingMode">CaptureFacingMode</dfn>
+          enumeration is used to express the <a>preferred facing mode</a>. The
+          semantics of its keywords mirror the similarly named keywords defined
+          in <a><code>VideoFacingModeEnum</code></a>.
+        </p>
+        <p class="note">
+          If the user agent is unable to support the <a>preferred facing
+          mode</a>, it can fall back to the implementation-specific default
+          facing mode.
+        </p>
+        <p>
+          The <a>capture</a> attribute is an <a>enumerated attribute</a> that
+          specifies the <a>preferred facing mode</a> for the <a>media capture
+          mechanism</a>. The attribute's keywords are <dfn data-dfn-for=
+          "CaptureFacingMode">user</dfn> and <dfn data-dfn-for=
+          "CaptureFacingMode">environment</dfn>, which map to the respective
+          states <var>user</var> and <var>environment</var>. The <a>missing
+          value default</a> is the <var>environment</var> state. The <a>invalid
```

Comment:
Added implementation-specific state in https://github.com/w3c/html-media-capture/pull/6/commits/5679164ddb3088c18efe42d9bce991f3e3551fe9

@miguelao PTAL to ensure this matches what you had in mind.